collections/_article/rendering-acyclic-graph.md

```diff
@@ -0,0 +1,279 @@
+---
+title: "GPU synchronization in Godot 4.3 is getting a major upgrade"
+excerpt: "Say goodbye to placing barriers, hello to the new acyclic graph"
+categories: ["progress-report"]
+author: Darío Banini
+image: /storage/blog/covers/todo.webp
+date: 2024-02-14 9:00:00
+---
+
+Since the introduction of Godot 4, [Rendering Device](https://docs.godotengine.org/en/stable/classes/class_renderingdevice.html) has been the backbone of the Forward+ and Mobile renderers. Making APIs that are both easy to use and flexible enough to cover all the features users want in a game engine is a very difficult task. As far as GPU synchronization is concerned, the high level of detail expected by APIs like Vulkan or D3D12 compared to its predecessors is worthy of a few blog posts of its own. This article will try to keep the topic of synchronization as brief as possible to focus on the problems solved within Godot instead.
```

Comment:
```suggestion
Since the introduction of Godot 4, [Rendering Device](https://docs.godotengine.org/en/stable/classes/class_renderingdevice.html) has been the backbone of the Forward+ and Mobile renderers. Making APIs that are both easy to use and flexible enough to cover all the features users want in a game engine is a very difficult task. As far as GPU synchronization is concerned, the high level of detail expected by APIs like Vulkan or Direct3D 12 compared to their predecessors is worthy of a few blog posts of its own. This article will try to keep the topic of synchronization as brief as possible to focus on the problems solved within Godot instead.
```

I know it's linked, so someone can press it for more info, but I think it would be nice to say "the Rendering Device API" or "the Rendering Device class" or similar in its first mention. Also, should it be RenderingDevice with no space?

I also found it slightly jarring that GPU synchronization is mentioned in the middle of the paragraph without any explanation as to what that is.

Reply:
I switched around between both and Rendering Device with the space looked better to me.

I agree the mention of synchronization isn't even necessary on this paragraph. I think explaining the concept would detract from the purpose of giving a very short overview of what the article is about so readers know if they're interested in diving into the topic... so I think removing the mention actually makes the paragraph better and works for the same purpose.